developer_manual/basics/storage/database.rst

```diff
@@ -45,6 +45,33 @@ Inside your database layer class you can now start running queries like:
     }
 
 
+Transactions
+------------
+
+Database operation can be run in a nested transaction to commit or roll back a group of changes in an atomic fashion.
```

Comment:
operation**s**

What is the significance of *nested*? Can I put a transaction into another transaction?

Reply:
> Can I put a transaction into another transaction?

Yes. You can have multiple levels.

Reply:
But I will drop this. It's unnecessarily confusing :)